chainregistry.go

```diff
@@ -30,15 +30,18 @@ import (
 
 const (
 	// TODO(decred) verify these amounts
```

Comment:
this todo has done?

Reply:
We haven't done any community-wide discussion regarding the minimum htlc amounts, timelocks, fees and feerates, etc so I'm leaving this as is for the moment until we get more usage out of our LN and can make a better informed decision on changing or leaving these as is.